iib/workers/tasks/build_merge_index_image.py

```diff
@@ -0,0 +1,278 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+import logging
+import tempfile
+
+from iib.exceptions import IIBError
+from iib.workers.api_utils import set_request_state
+from iib.workers.tasks.build import (
+    _add_label_to_index,
```

Comment:
Private functions really shouldn't be imported, but I'll look the other way on this one :stuck_out_tongue: